### PR TITLE
[doc]Add FAQ about how to reload your own plugin

### PR DESCRIPTION
**Pull request by Miss-you**
Add FAQ about how to reload your own plugin

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

**Comment by Miss-you** [comment]
I have a question, why can't I use the `apisix reload` command? Or that the command currently has a bug?

File: FAQ.md [hunk 1]
```diff
@@ -172,3 +172,17 @@ Steps:
...
+The Apache APISIX plugin supports hot reloading. If your APISIX node has the Admin API turned on, then for scenarios such as adding / deleting / modifying plugins, you can hot reload the plugin by calling the HTTP interface without restarting the service.
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT
+```
+
+If your APISIX node does not open the Admin API, then you can manually load the plug-in by reloading openresty.
+
+```shell
+openresty -p /usr/local/apisix -c /usr/local/apisix/conf/nginx.conf -s reload
```
Comment (membphis) [inline FAQ.md:187]: `apisix reload` is better. use `CLI` is simpler.

File: FAQ_CN.md [hunk 2]
```diff
@@ -123,3 +123,17 @@ https://github.com/iresty/lua-resty-radixtree#operator-list
...
+
+## 如何加载自己编写的插件
+
+Apache APISIX 的插件支持热加载，如果你的 APISIX 节点打开了 Admin API，那么对于新增/删除/修改插件等场景，均可以通过调用 HTTP 接口的方式热加载插件，不需要重启服务。
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT
+```
+
+如果你的 APISIX 节点并没有打开 Admin API，那么你可以通过手动 reload openresty 的方式加载插件。
```
Comment (moonming) [inline FAQ_CN.md:135]: `reload openresty` -> `reload APISIX`

**Comment by membphis** [comment]
> I have a question, why can't I use the `apisix reload` command? Or that the command currently has a bug?

I think we should advise user to use `apisix reload`.

File: FAQ.md (hunk 1 above)
Comment (Miss-you) [reply FAQ.md:187]: I originally wanted to use the `apisix reload` command, which ultimately triggered the process of `openresty reload nginx.conf`.

But I tried this command, I found that reload did not take effect, so I used `openresty -p /usr/local/apisix -c /usr/local/apisix/conf/nginx.conf -s reload`.

**Pull request merged**